src/main/java/core/basesyntax/MainApp.java

```diff
@@ -1,5 +1,16 @@
 package core.basesyntax;
 
 public class MainApp {
+    public static void main(String[] args) {
 
+        Machine truck = new Truck();
```

Comment:
![image](https://github.com/mate-academy/jv-oop/assets/20326858/2ea95222-d9b5-4141-a2f0-2d2b8dd8121e)

Reply:
Line No 5 is still empty. Do not begin methods with empty line.